package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chauffeur-prive/node-amqp-bus",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
This is not a major release from my opinion, more a bug fixing and possibly connectivity issue